### DISCUSSION TITLE
WinMerge folder comparison shows the contents of both the folders only on left side

### DISCUSSION THREAD

**Discussion by itsraghz** [WinMerge/winmerge#744] [General]
Hi,

 The WinMerge folder comparison shows the contents of both the folders only on the left side. I tried managing the settings but nothing to avail. 
  Any help will be appreciated.

**Comment by sdottaka** [comment]
WinMerge displays the comparison results of the left and right folders in one pane.

There is no mode to show folders in two panes.
